.github/workflows/update_licenses.yaml

```diff
@@ -0,0 +1,67 @@
+name: Update licenses
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  update_licenses:
+    strategy:
+      # TODO: find a way to do this on ARM MacOS
+      matrix:
+        os: [ubuntu-20.04, macos-latest-xl]
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    steps:
+      # Mask internal URLs if logged
+      - name: Add masks
+        run: |
+          echo "::add-mask::${{ secrets.INTERNAL_PYPI_URL_FOR_MASK }}"
+          echo "::add-mask::${{ secrets.INTERNAL_REPO_URL_FOR_MASK }}"
+
+      # Replace default archive.ubuntu.com from docker image with fr mirror
+      # original archive showed performance issues and is farther away
+      - name: Docker container related setup and git installation
+        run: |
+          TZ=Europe/Paris
+          echo "TZ=${TZ}" >> "$GITHUB_ENV"
+          ln -snf /usr/share/zoneinfo/${TZ} /etc/localtime && echo ${TZ} > /etc/timezone
+          sed -i 's|^deb http://archive|deb http://fr.archive|g' /etc/apt/sources.list
+          apt update && apt install git git-lfs -y
+
+      - name: Checkout Code
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+
+      - name: Install dependencies
+        if: ${{ matrix.os == "ubuntu-20.04" }}
+        run: |
+          ./script/make_utils/setup_os_deps.sh --linux-install-python
+          make setup_env
+
+      - name: Install dependencies
+        if: ${{ matrix.os != "ubuntu-20.04" }}
+        run: |
+          ./script/make_utils/setup_os_deps.sh
+          make setup_env
+
+      - name: Set version
```

Comment:
wrong step name (you only build licenses here)

Reply:
oups yes bad copy-paste